platform/src/default.rs

```diff
@@ -0,0 +1,153 @@
+// Licensed under the Apache-2.0 license
+
+use crate::{Platform, PlatformError, MAX_CHUNK_SIZE};
+use core::cmp::min;
+
+pub struct DefaultPlatform;
+
+pub const TEST_CERT_CHAIN: [u8; 2048] = [
```

Comment:
nit: Could you add a comment with how this is generated (in case we need to re-generate it)